migration/sql-files/081-queries.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE queries (
+    created_at timestamp with time zone,
+    updated_at timestamp with time zone,
+    deleted_at timestamp with time zone,
+    id uuid primary key DEFAULT uuid_generate_v4() NOT NULL,
+    title text NOT NULL CHECK(title <> ''),
+    fields jsonb,
```

Comment:
Should be `NOT NULL`?

Reply:
yes that is a good thing to have, added in https://github.com/fabric8-services/fabric8-wit/pull/1824/commits/2c7f2d8b82dfd68ccfa159a7e6373bfe991266c4